pkg/tbtc/dkg_loop.go

```diff
@@ -15,6 +15,32 @@ import (
 	"golang.org/x/exp/slices"
 )
 
+const (
+	// dkgAttemptAnnouncementDelayBlocks determines the duration of the
+	// announcement phase delay that is preserved before starting the
+	// announcement phase.
+	dkgAttemptAnnouncementDelayBlocks = 1
+	// dkgAttemptAnnouncementActiveBlocks determines the duration of the
+	// announcement phase that is performed at the beginning of each DKG
+	// attempt.
+	dkgAttemptAnnouncementActiveBlocks = 5
+	// dkgAttemptProtocolBlocks determines the maximum block duration of the
+	// actual protocol computations.
+	dkgAttemptMaximumProtocolBlocks = 150
+	// dkgAttemptCoolDownBlocks determines the duration of the cool down
+	// period that is preserved between subsequent DKG attempts.
+	dkgAttemptCoolDownBlocks = 5
+)
```

Comment:
`dkgRetryLoop struct` still has `announcementDelayBlocks` and `announcementActiveBlocks`. Should we remove them?